lib/puppetlabs_spec_helper/rake_tasks.rb

```diff
@@ -116,14 +117,14 @@ def fixtures(category)
       elsif opts.instance_of?(Hash)
         target = "spec/fixtures/modules/#{fixture}"
         real_source = eval('"'+opts["repo"]+'"')
-        result[real_source] = { "target" => target, "ref" => opts["ref"], "branch" => opts["branch"], "scm" => opts["scm"], "flags" => opts["flags"]}
+        result[real_source] = { "target" => target, "ref" => opts["ref"], "branch" => opts["branch"], "scm" => opts["scm"], "flags" => opts["flags"], "dirpath" => opts["dirpath"]}
```

Comment:
please use a descriptive name for your new option that would help a newcomer to understand what it is about. e.g. `subdir`